packages/pluggableWidgets/line-chart-native/src/components/LineChart.tsx

```diff
@@ -69,9 +69,7 @@ export function LineChart(props: LineChartProps): ReactElement | null {
                 lineStyle === "lineWithMarkers" ||
                 (lineStyle === "custom" && seriesStyle?.markers?.display && seriesStyle.markers.display !== "false") ? (
                     <VictoryScatter data={dataPoints} style={seriesStyle?.markers} size={seriesStyle?.markers?.size} />
-                ) : (
-                    undefined
-                );
+                ) : undefined;
```

Comment:
I think prettier prefers the previous revision. Has this changeset been run through prettier?

Reply:
new prettier changed patterns apparently.